packages/eservice-descriptors-archiver/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "pagopa-interop-eservice-descriptors-archiver",
+  "private": true,
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability descriptors archiver: a consumer service that archives descriptors on agreement archiviation events",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "test:it": "vitest integration",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "build": "tsc",
+    "trim-open-api": "mkdir -p ./src/model/generated && openapi-endpoint-trimmer -i '../catalog-process/open-api/catalog-service-spec.yml' -o './src/model/generated/spec.yaml' -p '/eservices/{eServiceId}/descriptors/{descriptorId}/archive'",
```

Comment:
Using `openapi-endpoint-trimmer` to trim the openapi file keeping only the route we need to archive the descriptor - as also recommended in the `openapi-zod-client` docs: https://github.com/astahmer/openapi-zod-client?tab=readme-ov-file#tips